INSTALL.md

```diff
@@ -46,84 +35,157 @@ platforms.
 | OpenSSL | 1.0.2 | Crypto functions for Google Cloud Storage authentication |
 
-Note that these libraries may have complex dependencies themselves, the
-following instructions include steps to install these dependencies too.
+Note that these libraries may also depend on other libraries. The following
+instructions include steps to install these indirect dependencies too.
+
 When possible, the instructions below prefer to use pre-packaged versions of
-these libraries. In some cases the packages do not exist, or the package
-versions are too old. If this is the case, the instructions describe how you
-can manually download and install these packages. 
+these libraries and their dependencies. In some cases the packages do not exist,
+or the package versions are too old to support `google-cloud-cpp`. If this is
+the case, the instructions describe how you can manually download and install
+these dependencies.
+
+## Table of Contents
+
+- [Fedora 29](#fedora-29)
+- [openSUSE (Tumbleweed)](#opensuse-tumbleweed)
+- [Ubuntu (18.04 - Bionic Beaver)](#ubuntu-1804--bionic-beaver)
+- [CentOS 7](#centos-7)
 
 ### Fedora (29)
 
+Install the minimal development tools:
+
+```bash
+sudo dnf makecache && \
+sudo dnf install -y cmake gcc-c++ git make openssl-devel pkgconfig zlib-devel
+```
+
 Fedora includes packages for gRPC, libcurl, and OpenSSL that are recent enough
-for `google-cloud-cpp`. Install these libraries and the necessary development
-tools:
+for `google-cloud-cpp`. Install these packages and additional development
+tools to compile the dependencies
 
 ```bash
-dnf makecache
-dnf install -y automake cmake gcc-c++ git grpc-devel grpc-plugins \
-        libcurl-devel make openssl-devel pkgconfig protobuf-compiler tar wget \
-        which zlib-devel
+sudo dnf makecache && \
+sudo dnf install -y grpc-devel grpc-plugins \
+        libcurl-devel protobuf-compiler tar wget zlib-devel
 ```
 
 #### crc32c
- 
-There is no Fedora package for this library. To install use:
+
+There is no Fedora package for this library. To install it, use:
 
 ```bash
 cd /var/tmp/build
 wget -q https://github.com/google/crc32c/archive/1.0.6.tar.gz
 tar -xf 1.0.6.tar.gz
-cd crc32c-1.0.6
+cd /var/tmp/build/crc32c-1.0.6
 cmake \
       -DCMAKE_BUILD_TYPE=Release \
       -DBUILD_SHARED_LIBS=yes \
       -DCRC32C_BUILD_TESTS=OFF \
       -DCRC32C_BUILD_BENCHMARKS=OFF \
       -DCRC32C_USE_GLOG=OFF \
       -H. -B.build/crc32c
-cmake --build .build/crc32c --target install -- -j $(nproc)
-ldconfig
+sudo cmake --build .build/crc32c --target install -- -j $(nproc)
+sudo ldconfig
 ```
 
 #### google-cloud-cpp
 
-We can now compile and install `google-cloud-cpp`. Note that we use `pkg-config`
-to discover the options for gRPC and protobuf:
+We can now compile and install `google-cloud-cpp`. Note that we use
+`pkg-config` to discover the options for gRPC and protobuf:
 
 ```bash
-cd $HOME/google-cloud-cpp # or wherever you have extracted google-cloud-cpp
+cd /home/build/google-cloud-cpp
```

Comment:
This might be an issue for users. Could you change back to `$HOME`?

Reply:
Fixed.